### PR TITLE
qli chapter 02 exercises

### PR DESCRIPTION
**Pull request by qianliok**

### CODE REVIEW

**Comment by qianliok** [comment]
hmm... it seems that we can only assign the pull request to one reviewer

**Comment by alexdeas** [comment]
Don't worry about assigning each pull request to someone, if you've got a review table everyone can just check that.

File: chapter02/qli/ex5.pl [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+use warnings;
+
+my $str=<STDIN>;
+chomp($str);
+
+my $num=<STDIN>;
+chomp($num);
+
+print "Ex Result:: \n";
+if ( $num && $num>0 ){
```
Comment (MarkWignall) [inline chapter02/qli/ex5.pl:12]: You decided to add some input validation?

Comment (JPeace2) [inline chapter02/qli/ex5.pl:12]: Why did you chomp and then add the new line back in?

Comment (qianliok) [inline chapter02/qli/ex5.pl:12]: Yes, thanks, I could remove both the chomp on the $str and the newline. As new line character will be picked by the input anyway.

Comment (qianliok) [inline chapter02/qli/ex5.pl:12]: @MarkWignall I don't think input validation is part of the requirements, I didn't add validation to all exercises too. I should have added more detailed program prompts.

**Pull request merged**